Refactor form generator to choose 1 or 4 pages

rhasselle-oddball opened the pull request:
- Refactor prompt to ask 1 or 4 pages for a starter, rather than blank, simple, complex
- Add pages using web component patterns
- Default to creating a form app if not selected

![image](https://github.com/user-attachments/assets/68dfdae1-85f8-4895-b3af-d7cee6b2d602)
![image](https://github.com/user-attachments/assets/f0842ecd-4b30-46fa-b1d4-94f7c00f477f)
![image](https://github.com/user-attachments/assets/9867dd3c-f277-4985-81b7-c2870f365f49)
![image](https://github.com/user-attachments/assets/8ddc0248-974f-4e94-af92-f8e01c967471)
![image](https://github.com/user-attachments/assets/f6fd4bfb-65ef-421f-a8f0-b11355ed0f5b)
---
![image](https://github.com/user-attachments/assets/565b043c-3175-495f-90ca-5dbcada2cb39)
![image](https://github.com/user-attachments/assets/f5d8d283-bd07-494d-9c78-11ac267c2672)

Mottie commented on generators/form/index.js:
```diff
@@ -2,7 +2,6 @@
 
 const TEMPLATE_TYPES = {
-  BLANK: 'BLANK',
   SIMPLE: 'SIMPLE',
   COMPLEX: 'COMPLEX',
```
Might as well rename these to something like `1-page` and `4-pages`? Because they display in the terminal

Mottie commented on generators/form/templates/pages/mailingAddress.js.ejs:
```diff
@@ -0,0 +1,26 @@
+  uiSchema: {
+    ...titleUI(
+      'Mailing address',
+      "We'll send any important information about your application to this address.",
```
We should be using the typographical apostrophe (`’`) here

Mottie reviewed (approved):
I kinda like having the blank template, but this is cool too. Otherwise, everything looks good to me!

rhasselle-oddball commented:
> I kinda like having the blank template, but this is cool too. Otherwise, everything looks good to me!

@Mottie ty for the feedback. Addressed comments. I think not having a blank is OK.  I figure it's easier to change/adjust 1 page with everything already setup correctly with recommend web component patterns, rather than adding it from scratch.

jamigibbs reviewed (approved):
Thanks again for working on this! Do you think any of the [platform forms library](https://depo-platform-documentation.scrollhelp.site/developer-docs/va-forms-library-getting-started-with-the-forms-li#VAFormsLibrary-GettingStartedwiththeFormsLibrary-CreateanewformapplicationwithYeomangenerator) docs will need to be updated for these changes?

rhasselle-oddball commented:
> Thanks again for working on this! Do you think any of the [platform forms library](https://depo-platform-documentation.scrollhelp.site/developer-docs/va-forms-library-getting-started-with-the-forms-li#VAFormsLibrary-GettingStartedwiththeFormsLibrary-CreateanewformapplicationwithYeomangenerator) docs will need to be updated for these changes?

@jamigibbs good call. I created a [ticket](https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team-forms/1581) for updating docs, after we've finished making changes in this repo (there are more coming in the next week)
